src/content/4/en/part4b.md

```diff
@@ -56,7 +56,7 @@ We specified the mode of the application to be <i>development</i> in the _npm ru
 There is a slight issue in the way that we have specified the mode of the application in our scripts: it will not work on Windows. We can correct this by installing the [cross-env](https://www.npmjs.com/package/cross-env) package with the command:
 
 ```bash
-npm install --save-dev cross-env
```

Comment:
Lets at least keep the `--save` flag here, otherwise it would cause issues when the code is ran.

Reply:
Isn't --save the default option ? I never used --save, only --save-dev, and I looked into the documentation and there isn't a --save option there https://docs.npmjs.com/cli/install

Reply:
Ok, it seems that `--save` flag has been removed in npm version 5, which was released in may 2017. I'm not sure about the usage statistics of earlier versions of npm.
In my opinion, having the flag here would be an implicit signal for saving, but on the other hand the flag has been removed.. Earlier (beginning of 2018) some students have had issues with packages not appearing in production, but no one should be running older versions of npm anymore, right? :D